iina/MainMenuActions.swift

```diff
@@ -154,8 +154,12 @@ extension MainMenuActionHandler {
 
   @objc func menuChapterSwitch(_ sender: NSMenuItem) {
     let index = sender.tag
-    player.playChapter(index)
-    let chapter = player.info.chapters[index]
+    guard let chapter = player.playChapter(index) else {
+      Logger.log("Cannot switch to chapter \(index) because it was not found! Will ignore request and reload chapters instead",
+                 level: .error, subsystem: player.subsystem)
```

Comment:
I think error is too strong for this situation. The user can trigger this. IINA recovers. There isn't an error that needs to be corrected. I would emit this as a debug level message.